misc/mouse.talon

```diff
@@ -6,8 +6,10 @@ touch:
 	mouse_click(0)
 	# close the mouse grid if open
 	user.grid_close()
```

Comment:
This is added because it's quite hard to stop a right drag

Without this only `righty` and `right drag` stop it whereas users are likely used to `touch` stopping drag commands since `touch` stops the left drag command